react/src/components/SessionUsageMonitor.tsx

```diff
@@ -0,0 +1,260 @@
+                      1,
+                    ),
+                  )}
+                  strokeLinecap="butt"
+                  size={{ height: 20 }}
+                  strokeColor={token.colorPrimary}
+                ></Progress>
+              </Flex>
+              <Typography.Text>
+                {`(mem) ${
+                  iSizeToSize(sortedLiveStat?.mem?.current, 'g')?.numberUnit
+                }iB
+                /
+                ${
+                  iSizeToSize(sortedLiveStat?.mem?.capacity, 'g')?.numberUnit
+                }iB`}
+              </Typography.Text>
+            </Flex>
+          </Col>
+        ) : null}
+        {/* accelerator type is in the format of `[device_name]_[util|mem]` */}
+        {_.map(sortedLiveStat, (value, key) => {
+          const device = key.split('_')?.[0] as keyof typeof resourceTypeMap;
+
+          return _.includes(
+            _.keysIn(_.omit(resourceTypeMap, 'cpu', 'mem')),
+            device,
+          ) ? (
+            <>
+              {/* only show accelerator's util and mem */}
+              {(_.includes(key, 'util') || _.includes(key, 'mem')) && (
+                <Col
+                  span={_.get(gridColSpanMap, col)}
+                  style={{ marginBottom: token.marginXS }}
+                >
+                  <Flex direction="column" align="end">
+                    <Flex
+                      align="start"
+                      gap={token.marginSM}
+                      style={{ width: '100%' }}
+                    >
+                      <ResourceTypeIcon
+                        type={resourceTypeMap[device]}
+                        showTooltip={true}
+                      />
+                      <Progress
+                        showInfo
+                        status="normal"
+                        percent={_.toNumber(
+                          toFixedFloorWithoutTrailingZeros(
+                            sortedLiveStat?.[device]?.pct || 0,
+                            1,
+                          ),
+                        )}
+                        strokeLinecap="butt"
+                        size={{ height: token.sizeMD }}
+                        strokeColor={token.colorPrimary}
+                      ></Progress>
+                    </Flex>
+                    <Typography.Text>
+                      {_.includes(key, 'mem') &&
+                        `${device} (mem) ${iSizeToSize(sortedLiveStat?.mem?.current, 'g')?.numberUnit}iB / ${iSizeToSize(sortedLiveStat?.mem?.capacity, 'g')?.numberUnit}iB`}
+                    </Typography.Text>
+                  </Flex>
+                </Col>
+              )}
+            </>
+          ) : null;
+        })}
+      </Row>
+      <Flex justify="end" style={{ width: '100%' }}>
+        <Typography.Text>
+          {`I/O Read: ${iSizeToSize(sortedLiveStat?.io_read?.current, 'g')?.numberUnit}B / Write: ${iSizeToSize(sortedLiveStat?.io_write?.current, 'g')?.numberUnit}B`}
```

Comment:
The iSizeToSize function is for binary size conversion, not for I/O. Please check the original version and ensure that the display matches it. See the reference here: https://github.com/lablup/backend.ai-webui/blob/main/src/components/backend-ai-session-list.ts#L4318-L4326

Reply:
In that code, the API request value comes in as a MB value, but not in ComputeSessionNode, so I modified it to show it in MB, the same as the session list usage column, using `convertDecimalSizeUnit`.